### DISCUSSION TITLE
Does anyone know how to use a SX1262 to recive POCSAG messages when the DIO2 is set to function as RF switch?

### DISCUSSION THREAD

**Discussion by tklecka** [jgromes/RadioLib#668] [Q&A] [answered]
Hey,
i recently bought a Heltec Wifi ESP32 Lora board (ESP32-S3 / SX1262) and wanted to receive POCSAG messages.
Looking at the `Pager_Receive` example i saw that the DIO2 is needed to receive digital data on.
But the DIO2 is set to function as RF switch?
Any ideas how i can receive POCSAG messages?

**Comment by jgromes** [comment, marked as answer]
Unfortunately, on SX126x, you can't - it's not listed in the Pager_Receive example.

The reason is that SX126x doesn't allow "direct mode" of operation, in which the demodulated data are directly output in its DIO pins, which is needed for reception of the very long Pager packets.

**Comment by tklecka** [comment]
Yeah, that's what I thought...
Thanks
